package days19;

import java.io.IOException;
import java.util.Random;

public class Ex01 {
	
	public static void main(String[] args)
			   throws Exception{
		// p 448 확인문제
		//System.in.read();
		//throw new  ArrayIndexOutOfBoundsException();
		Random rnd = new Random();
		
		System.out.println( rnd.nextInt(20)+1); //
		/*
		try {
			method1(); 
		} catch (ClassNotFoundException e) {
		} catch( Exception e ) {
			
		}
		*/
		
	}
	
}
